packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -512,6 +519,9 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  /// The icon used for this button.
+  final Icon icon;
+
```

Comment:
we should add some more depth to the docs for `child` and `icon` that explains what each does and how it relates to the other and that `icon` has a default if neither is provided.